docs/configure_sensors.md

```diff
@@ -0,0 +1,59 @@
+# Configure Sensors
+To navigate to the configure sensors page, click on the "Configure Sensors" icon on the left side of the webpage. The configure sensors page is broken into two components: adding new sensors and managing existing sensors. 
+
+## Adding New Sensors
+In order to add a sensor to your Mercury instance, you must use the following Add Sensor Form, which can be found at the top of the configure sensors page.
+![Add Sensor Form](imgs/add_sensor_form.png)
+
+- Sensor Name
+  - To add a new sensor, first type a name into the sensor name box. 
+  - Restrictions:
+    - All sensors must have non-empty names.
+    - You cannot have two sensors with the same name in the database. Note that all sensor names are stored in lowercase in the database, so this restriction is NOT case sensitive.
+- Fields
+  - Once you decided on the sensor name, it's time to give your sensor some fields. For some sensors, such as a temperature sensor, one field is enough. In this case, it is perfectly acceptable to name your field the same as your sensor name.
+  - For other sensors, such as a triple-axis accelerometer, multiple fields are required. In this case, you can click the green "+" button at the bottom of the Action column to add a new field. 
+  - Notice that each field row also has an "X" in its Action column; if you accidentally inserted too many fields, you can always delete any of the fields with this "X" button. 
```

Comment:
@dandipietrantonio @Ray310 can we put both the "X" and "+" buttons right at the end of each row? Rather than the "X" at the end of the row and the "+" below the row? I think it will look better and we should get the change in before the documentation is finalized if possible